front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/index.tsx

```diff
@@ -66,9 +67,9 @@ const CustomPageSettingsForm = ({
 
   const onFormSubmit = async (formValues: FormValues) => {
     try {
-      onSubmit(formValues);
+      await onSubmit(formValues);
```

Comment:
Based on the type of onSubmit, this await has no effect. I pushed a commit to fix that.